lib/cb/clients/recommendation.rb

```diff
@@ -65,12 +65,15 @@ def self.for_user(*args)
       def self.for_company(company_did)
         my_api = Cb::Utils::Api.instance
         json_hash = my_api.cb_get(Cb.configuration.uri_recommendation_for_company, query: { CompanyDID: company_did })
-
         jobs = []
         if json_hash.key?('Results')
           if json_hash['Results'].key?('JobRecommendation')
-            json_hash['Results']['JobRecommendation']['Jobs'].each do |cur_job|
-              jobs << Models::Job.new(cur_job)
+            if json_hash['Results']['JobRecommendation'].key?('Jobs')
+              if json_hash['Results']['JobRecommendation']['Jobs'].key?('CompanyJob')
+                json_hash['Results']['JobRecommendation']['Jobs']['CompanyJob'].each do |cur_job|
+                  jobs << Models::Job.new(cur_job)
+                end
+              end
             end
```

Comment:
It will attempt to do the lookup for each key in this way, but because these are hashes, the access time is `O(1)`, so that's relatively meaningless compared to us checking each key one at a time. As bonus, `{}.fetch('somekey', {})` should be super fast because there are no keys to actually fetch against. All in all, I think using `fetch` might be a millisecond faster at best and the same speed at worst compared to the nested if statements you have today.